osgp/shared/osgp-kafka-distributionautomation/src/main/resources/ScadaMeasurementPublishedEventMessage.avsc

```diff
@@ -0,0 +1,195 @@
+{
+  "type" : "record",
+  "name" : "Message",
+  "namespace" : "com.alliander.data.scadameasurementpublishedevent",
+  "doc" : "default message format for Alliander Messaging*",
+  "fields" : [ {
+    "name" : "messageId",
+    "type" : {
+      "type" : "fixed",
+      "name" : "MessageId",
+      "namespace" : "com.alliander.messaging",
+      "size" : 16
+    },
+    "doc" : "contains a version 4 UUID generated by the process constructing the message"
+  }, {
+    "name" : "createdDateTime",
+    "type" : "long",
+    "doc" : "the timestamp at which the message is created in the message creating process as the number of milliseconds from the unix epoch, 1 January 1970 00:00:00.000 UTC"
+  }, {
+    "name" : "producerId",
+    "type" : "string",
+    "doc" : "identifier of the message producing process*"
+  }, {
+    "name" : "signature",
+    "type" : [ "null", "bytes" ],
+    "doc" : "2048bits RSA encrypted SHA-256 hash over the avro binary representation of the complete message assuming the signature field is set to null"
+  }, {
+    "name" : "payload",
+    "type" : {
+      "type" : "record",
+      "name" : "ScadaMeasurementPublishedEvent",
+      "doc" : "measurement data as published by the SCADA system. This class (and its topic in Kafka) will be replaced by GridMeasurementPublishedEvent",
```

Comment:
Does this mean the ScadaMeasurementPublishedEvent record is only temporary and will be replaced by GridMeasurementPublishedEvent at a certain point in time?

Reply:
It may be so.